.github/workflows/beagle-bone-blue-binaries.yml

```diff
@@ -0,0 +1,101 @@
+name: beagle-bone-blue-binaries
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5,17 * * *'
+jobs:
+  latest:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      #- run: sudo apt update
+      #- run: sudo apt install git
+      - run: git clone https://github.com/ardupilot/ardupilot.git
```

Comment:
why rerun the git clone ? you are already on a fresh clone

Reply:
Right, needs to be removed also. I used this config in my own repo https://github.com/drtrigon/ardupilot-beagle-bone-blue-binaries and forgot to remove that line.